src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Controllers/AuthenticationController.cs

```diff
@@ -231,6 +235,7 @@ public async Task<ActionResult> AuthenticateUser([FromQuery] string goTo, [FromQ
         /// </summary>
         /// <returns>Ok response with the refreshed token appended.</returns>
         [Authorize]
+        [Produces("text/plain")]
```

Comment:
This doesn't seem to be enough to affect the OpenAPI specification. 
Should we try to add the ProducesReponseTypeAttribute as well?

Reply:
You're absolutely correct. I have no idea how I came to the conclusion that this was adequate here.

Reply:
Could you please add a ProducesReponseTypeAttribute ? I hope it's faster an easier for you as the source branch is from your fork.

Reply:
Yea, I'll fix it when my local is ready to change branches.

Reply:
Done